server/views/licenceConditions/standard.pug

```diff
@@ -19,6 +20,52 @@ block content
         for condition in conditions
           li.midPaddingBottom #{condition.text}
 
+
+  h2.heading-large Additional conditions
+  div.pure-u-1.pure-u-md-2-3
+    - var results = false
+    ol.numberedList
+      for condition in licenceConditions
+          if condition.group !== undefined && condition.group !== 'Post-sentence supervision only' && condition.group !== 'Bespoke'
+            - results = true
+              li.midPaddingBottom 
+                for item in condition.content
+                  span  #{item.text}
+                  if item.variable
+                    span #{item.variable}
+    if results === false
+      | No additional conditions selected
+
+  h2.heading-large Post sentence supervision conditions
+  div.pure-u-1.pure-u-md-2-3
+    - var results = false
+    ol.numberedList
+      for condition in licenceConditions
+          if condition.group === 'Post-sentence supervision only' && condition.group === 'Post-sentence supervision only'
```

Comment:
doubles up condition?

Reply:
oops, I was tinkering with mixins that took in 2 args because the additional and  PSS conditions needed them. Then I ditched the idea of a mixin but forgot to take out the above.
Done.